### ISSUE TITLE
twitter bootstrap breaks context menu

### ISSUE THREAD

**Issue by dude8** [josdejong/jsoneditor#280]
if jsoneditor is used on a page which includes also [twitter bootstrap](http://getbootstrap.com) the context menu breaks.

![image](https://cloud.githubusercontent.com/assets/11646216/13888083/ae84cd90-ed3f-11e5-83ab-19e552b58b59.png)

This is due to`box-sizing: border-box;` which will be applied to all elements by bootstrap.
It can be fixed by explicitly set `box-sizing` to `content-box` for the menu.
```css
  .jsoneditor-menu { /* bootstrap fix */
    -webkit-box-sizing: content-box;
    -moz-box-sizing: content-box;
    box-sizing: content-box;
  }
```

**Comment by josdejong** [comment]
Argh! That bootstrap keeps breaking stuff in the most unexpected places. So intrusive.

Thanks for bringing this up, will fix it.

**Comment by josdejong** [comment]
Fixed in v5.2.0
